### PR TITLE
Display permissions for use cases for the current user

### PR DESCRIPTION
**Pull request by gusevda**
Closes https://github.com/giantswarm/roadmap/issues/1068.

This PR is a feature branch for the permissions overview page, and contains the following changes:

- [x] https://github.com/giantswarm/roadmap/issues/1051
- "Inspect permissions" page scaffold was added;
- a user menu item which allows a user to navigate to the new page was added;

Screenshot 2022-04-27 at 11 39 55

- [x] https://github.com/giantswarm/happa/pull/3399
- for screenshots and details, please see the above PR's description

### CODE REVIEW

**Review by kuosandys — COMMENTED** [review]
Nice 👍 As discussed, we'll build the PR for the page contents on this branch and merge everything when finished.

**Comment by kuosandys** [comment]
Reviewed the UI today with Marian, here's the feedback for changes to be made:

Page:
- Decrease the horizontal margins for the "Inspect Permissions" page, allowing more width for the page contents

Organizations:
- Increase space between columns in the "Organizations" tab to help with information grouping
- Display organization names in font weight `bold`

Categories:
- Display all use case categories as collapsed initially
- Order use case categories alphabetically

Statuses:
- Align all status icons to the left (including "Partial" status)
- Change font color of "Partial" status and icon to `#ccc`
- Align font size of permissions statuses to the same size as column headers and use case names (font size `15px`)

Further suggestion (i.e. we can try this out, though not completely necessary at this point):
- in "Organizations" tab, highlight column on hover

After these changes, we can merge this branch and close the parent issue 👍

**Comment by gusevda** [comment]
Decided with Marian to leave these changes for future improvements:

- Decrease the horizontal margins for the "Inspect Permissions" page, allowing more width for the page contents

- in "Organizations" tab, highlight column on hover

**Review by kuosandys — APPROVED** [review]
Looks great - LGTM 🎉

File: src/components/MAPI/permissions/Permissions.tsx
Comment (marians) [inline src/components/MAPI/permissions/Permissions.tsx:36]: I'd like to change the first sentence

> Here you get an overview of your RBAC permissions in the management cluster.

to

> Here you get an overview of your RBAC permissions in the management cluster, with regard to certain use cases.

**Review by marians — APPROVED** [review]
I found one little text change I'd like us to make at some point. Shouldn't block this PR. Let's get the good work out!

**Comment by kuosandys** [comment]
Since we haven't merged this yet, something just occurred to me: Should we be displaying this "Inspect Permissions" page for users not logged in via SSO (and thus using MAPI), right? There's currently no check for that.

**Pull request merged**
